packages/astro/src/@types/astro.ts

```diff
@@ -88,7 +88,7 @@ export type {
 
 export interface AstroBuiltinProps {
 	'client:load'?: boolean;
-	'client:idle'?: boolean;
+	'client:idle'?: IdleRequestOptions | boolean;
```

Comment:
Is the `IdleRequestOptions` type defined? I can't seem to find it.

Reply:
already defined in ./node_modules/typescript/lib/lib.dom.d.ts